package.json

```diff
@@ -43,11 +43,12 @@
 		"eslint-plugin-standard": "^4.0.0",
 		"ipfs-http-client": "^29.1.1",
 		"spawn-command": "0.0.2-1",
-		"watch": "^1.0.2"
+		"watch": "^1.0.2",
+		"ws": "^6.2.1"
```

Comment:
does this is a must ?

Reply:
The CI build failed without: https://github.com/daostack/subgraph/runs/112147150

Reply:
`ws` is a dependency of `subscriptions-transport-ws` and `web3`  and it is pulled automatically . there is no need to add that to package.json.
see : 
<img width="399" alt="Screen Shot 2019-04-28 at 0 05 45" src="https://user-images.githubusercontent.com/5994398/56855109-ebbbe700-6949-11e9-881b-c5ae39202258.png">

please update package-lock.json  : `rm package-lock.json` and `npm i`

Reply:
That did the trick, thank you!